tests/linearGaussian_vector_field_test.py

```diff
@@ -407,6 +587,81 @@ def test_vector_field_iid_inference(
     )
 
 
+# TODO: Needs to fine-tune hyper-parameters
+# TODO: for jac_gauss after first sampling GradTrackingTensor appear
```

Comment:
Mhh, crazy not sure what happens there. Since when did PyTorch introduce Tacer Tensors :o  Might be something due to the new pytorch version.